client/pom.xml

```diff
@@ -147,6 +158,12 @@
             <artifactId>hibernate-core</artifactId>
             <scope>compile</scope>
         </dependency>
+        <!--<dependency>
```

Comment:
Sorry for nitpicking, but if this is commented out, why add it in the first place?

Reply:
You are right, commented it out to test, if we need it to compile and forgot to remove it.